cmd/controller-bootstrap/command/apiutil.go

```diff
@@ -13,7 +13,177 @@
 
 package command
 
-// TODO: getServiceResources infers aws-sdk-go to fetch the service metadata and custom resource names
-func getServiceResources() error {
-	return nil
+import (
+	"errors"
+	"fmt"
+	"github.com/gertd/go-pluralize"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
+)
```

Comment:
```suggestion
import (
	"errors"
	"fmt"
	"io/ioutil"
	"os"
	"path/filepath"
	"sort"
	"strings"

	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
	"github.com/gertd/go-pluralize"
)
```

if you do not have goimports file-watcher, I recommend adding it to your IDE. It will help sort your imports.